packages/config/config/devices/0x031e/lzw30.json

```diff
@@ -238,16 +242,48 @@
 				}
 			]
 		},
-		"8": {
-			"label": "LED Strip Effect",
-			"description": "Calculated sum of parameters as per overview section",
-			"valueSize": 4,
+		"13": {
+			"label": "Special Load Type",
```

Comment:
And here.
It can then be removed from the descriptions
```
{
  "$if": "firmwareVersion >= 1.17",
  "label": "Special Load Type",
   ......
}
```